test/unit/gateways/cashnet_test.rb

```diff
@@ -69,7 +69,7 @@ def test_add_creditcard
     @gateway.send(:add_creditcard, result, @credit_card)
     assert_equal @credit_card.number, result[:cardno]
     assert_equal @credit_card.verification_value, result[:cid]
-    assert_equal '0915', result[:expdate]
+    assert_equal '0916', result[:expdate]
```

Comment:
Can we use `Time.year.now` and prevent future breakage? Goes for most/all other changes, too.